### DISCUSSION TITLE
Mobile number based authentication protocol.

### DISCUSSION THREAD

**Discussion by rakneo** [saleor/saleor#6303] [Ideas]
As of now lot of e-commerce website have adopted to give users login through mobile number and SMS based OTP. Is there any way to implement this in saleor.

**Comment by karolkielecki** [comment]
@rakneo Hey! Thanks for asking. Well, our user authentication is an email. We have a mutation for external authentication. We haven't published this chapter yet, but you can follow this implementation when building your solution: https://github.com/mirumee/saleor-docs/pull/182/files

**Comment by bisverse** [comment]
Is there any way to sign up using mobile number (no email) and later verify user with otp. I looked into oidc documentation but it requires email as response.

**Comment by timuric** [comment]
Please refer to https://docs.saleor.io/docs/3.x/developer/available-plugins/openid-connect#mutation
